tests_e2e/requirements.txt

```diff
@@ -10,4 +10,3 @@ cryptography
 distro
 junitparser
 msrestazure
-python-dotenv
```

Comment:
I removed this dependency. We are over-using the environment to pass data across different components. We need to reduce the use of environment variables, so this module should not be needed.

Reply:
Use of environment variables should be restricted to the interface with the Azure pipelines and probably a few (if any) places. If used for other things other than communication with the pipeline, we need to be very explicit about what variables we are  using. There should not be a need for this module to save and restore the entire environment.